### ISSUE TITLE
[Feature] For Drugsales

### ISSUE THREAD

**Issue by maguhd** [DiscworldZA/gta-resources#18]
**Is your feature request related to a problem? Please describe.**
No.

**FEATURE ADD**
Adding zone only selling feature. Like create an area of 5-10 blocks or so, to sell. + different locations for every drug. 4 or 5 zones should be fine and a config to attribute the drug to which zone to sell.

I just found out about your project! Great work man! Keep it up! Thank you! You saved me a lot of time! Appreciate it!

**Comment by DiscworldZA** [comment]
Would this be areas inside the game where you can sell only? I can add this. I imagine like neighborhoods?

**Comment by maguhd** [comment]
Yeah ofc. Example: Sell weed only on Groove street neighbourhood. Meth in the ghettos. Coke on vinewood...

**Comment by DiscworldZA** [comment]
Ok you want to be able to limit drugs per neighborhood aswell. Ok cool that's a sick idea!

**Comment by maguhd** [comment]
Yes! Restric the whole map except those areas!

**Comment by DiscworldZA** [comment]
Cool ill get on it.
